articles/openshift/oshift-how-portworx-snapshots.md

```diff
@@ -0,0 +1,347 @@
+        matchLabels:
+          purpose: group-test" | oc create -f -
+    ```
+
+2. This will create a `VolumeSnapshot` object for each PVC that it matches. To find the name of the volume snapshots you can describe the `GroupVolumeSnapshot` object:
+
+    ```none
+    oc describe groupvolumesnapshot | grep "Volume Snapshot Name"
+    
+        Volume Snapshot Name:  test-groupsnapshot-group-snapshot-test-2-61d7ca5e-ee87-11e9-8422-fa163e52fd0e
+        Volume Snapshot Name:  test-groupsnapshot-group-snapshot-test-1-61d7ca5e-ee87-11e9-8422-fa163e52fd0e
+        Volume Snapshot Name:  test-groupsnapshot-group-snapshot-test-0-61d7ca5e-ee87-11e9-8422-fa163e52fd0e
+    ```
+
+3. To restore any of the PVCs, create a PVC object and insert the relevant snapshot name:
+
+    ```none
+    echo "apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: pvc-restore-1
+      namespace: test-snapshot-1
+      annotations:
+        snapshot.alpha.kubernetes.io/snapshot: <snapshot name>
+    spec:
+      accessModes:
+         - ReadWriteOnce
+      storageClassName: stork-snapshot-sc
+      resources:
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+### Cloud snapshots
+
+Portworx enables you to backup snapshots to cloud storage. We can provide a bucket for the cluster on our Cloud Storage service and set up a cloud credential that you can use. If you'd like to configure snapshots to be sent to your own storage, see <https://docs.portworx.com/reference/cli/credentials/#overview> for how to set this up. 
+
+> [!TIP]
+> To find the ID of the cloud credentials provided with your cluster run:
+> `PX_POD=$(kubectl get pods -l name=portworx -n kube-system -o jsonpath='{.items[0].metadata.name}')`
+> `kubectl exec $PX_POD -n kube-system -- /opt/pwx/bin/pxctl credentials list`
+
+1. Create a PVC to use for the cloudsnap:
+
+    ```none
+    echo "apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: test-cloudsnap
+      namespace: test-snapshot-1
+      annotations: 
+        volume.beta.kubernetes.io/storage-class: portworx-repl1
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+2. To create a cloud snapshot, create a `VolumeSnapshot` object with the `portworx/snapshot-type=cloud` annotation:
+
+    ```none
+    echo "apiVersion: volumesnapshot.external-storage.k8s.io/v1
+    kind: VolumeSnapshot
+    metadata:
+      name: test-cloudsnap
+      namespace: test-snapshot-1
+      annotations:
+        portworx/snapshot-type: cloud
+    spec:
+      persistentVolumeClaimName: test-cloudsnap" | oc create -f -
+    ```
+
+    You can also provide the cloud credentials for your chosen cloud storage in the `portworx/cloud-cred-id` annotation, however this is only necessary when there are multiple configured. In our Portworx clusters out of the box you won't need to specify this but it would look like the below:
```

Comment:
multiple whats configured?

Reply:
Multiple cloud storage destinations - should I reword it?

Reply:
I think that would make it clearer. The sentence didn't really make sense to me. So change to something like:

however, this is only necessary when there are multiple possible storage destinations.

Reply:
Updated :)